deduplication/src/main/java/gov/cdc/nbs/deduplication/merge/PatientMergeController.java

```diff
@@ -41,22 +40,34 @@ public PatientMergeController(
 
   @GetMapping
   public MatchesRequireReviewResponse getPotentialMatches(
-          @RequestParam(defaultValue = "0", name = "page") int page,
-          @RequestParam(defaultValue = "5", name = "size") int size,
-          @RequestParam(defaultValue = DEFAULT_SORT, name = "sort") String sort) {
+      @RequestParam(defaultValue = "0", name = "page") int page,
+      @RequestParam(defaultValue = "5", name = "size") int size,
+      @RequestParam(defaultValue = DEFAULT_SORT, name = "sort") String sort) {
     return matchesRequiringReviewResolver.resolve(page, size, sort);
   }
 
   @GetMapping("/{patientId}")
   public ResponseEntity<List<PersonMergeData>> getPotentialMatchesDetails(
-          @PathVariable("patientId") Long patientId) {
+      @PathVariable("patientId") Long patientId) {
     return ResponseEntity.ok(mergeGroupHandler.getPotentialMatchesDetails(patientId));
   }
 
-  @PostMapping("/group-no-merge")
-  public ResponseEntity<String> updateGroupNoMerge(@RequestBody GroupNoMergeRequest request) {
+
+  @PostMapping("/unmerge-all/{personId}") // Keep Separate
+  public ResponseEntity<String> unMergeAll(@PathVariable("personId") Long personId) {
+    try {
+      mergeGroupHandler.unMergeAll(personId);
+      return ResponseEntity.ok("Merge status updated successfully.");
+    } catch (Exception e) {
+      return ResponseEntity.status(500).body("Error updating merge status: " + e.getMessage());
+    }
+  }
+
+  @PostMapping("/unmerge_single_person/{personId}/{potentialPersonId}")
```

Comment:
Request: can we update the route for unmerge-all to be

DELETE ->`/{patientId}/` 
This will include the controller level base of `/merge`. So the full route would be `/merge/{patientId}`

and the route for removing a single entry from merge to be:

DELETE -> `/{patientId}/{removePatientId}`